### PR TITLE
Issuee 99

### PR DESCRIPTION
**Pull request by alizahir23**
This PR solves issue #99

**What does this PR do?**
adds the ability to select organisations and languages on initially signing in

### CODE REVIEW

File: src/components/Home/WelcomeComponent.jsx [hunk 1]
```diff
@@ -19,24 +18,33 @@ export default function WelcomeComponent() {
...
-    }
-    else
-    Router.push('/');
+    if (newUser.code === undefined) {
+      changeUser(
+        newUser.user.displayName,
+        newUser.user.email,
+        newUser.user.uid
+      );
+      if (newUser.additionalUserInfo.isNewUser) {
```
Comment (irshadjsr21) [inline src/components/Home/WelcomeComponent.jsx:27]: We need to add this to Github login also.

File: src/components/TopLanguages.jsx [hunk 2]
```diff
@@ -1,14 +1,117 @@
...
+import React, { useState, createRef, useContext } from 'react';
 
-import styles from '../scss/lang.module.scss';
+import firebase from '../firebase';
+import styles from '../scss/org.module.scss';
+import UserContext from "./UserContext";
+
+export default function TopOrganisation() {
+  const { User } = useContext(UserContext);
+  const orgListRef = createRef();
```
Comment (irshadjsr21) [inline src/components/TopLanguages.jsx:10]: Avoid using `Ref`. We can achieve this with react hooks.

File: src/components/TopOrganisations.jsx [hunk 3]
```diff
@@ -1,14 +1,27 @@
...
+import Router from 'next/router';
+import React, { useState, useEffect, createRef, useContext } from 'react';
 
+import firebase from '../firebase';
 import styles from '../scss/org.module.scss';
+import UserContext from "./UserContext";
 
 export default function TopOrganisation() {
+  const { User } = useContext(UserContext);
   const orgListRef = createRef();
```
Comment (irshadjsr21) [inline src/components/TopOrganisations.jsx:10]: Same here, avoid using refs.

File: src/components/TopOrganisations.jsx [hunk 4]
```diff
@@ -1,14 +1,27 @@
...
+import React, { useState, useEffect, useRef, useContext } from 'react';
 
+import firebase from '../firebase';
 import styles from '../scss/org.module.scss';
+import UserContext from './UserContext';
 
 export default function TopOrganisation() {
-  const orgListRef = createRef();
+  const { User } = useContext(UserContext);
+  const orgListRef = useRef();
```
Comment (irshadjsr21) [inline src/components/TopOrganisations.jsx:10]: Avoid using `Ref`.

**Comment by alizahir23** [comment]
that scrollbar os for the entire page, shouldn't it be there?

**Pull request merged**